__tests__/app.test.js

```diff
@@ -220,6 +228,28 @@ describe("NC News Server", () => {
         });
     });
   });
+  describe("GET QUERY /api/articles returns articles with correct query", () => {
+    test("GET 200 : /api/articles?topic gives the articles with the queried topic", () => {
+      return request(app)
+      .get("/api/articles?topic=cats")
+      .expect(200)
+      .then(({ body }) => {
+        const {articles} = body
+        expect(articles).toHaveLength(1)
+        body.articles.forEach((article) => {
+          expect(typeof article.author).toBe("string");
+          expect(typeof article.title).toBe("string");
+          expect(typeof article.article_id).toBe("number");
+          expect(typeof article.topic).toBe("string");
+          expect(typeof article.created_at).toBe("string");
+          expect(typeof article.votes).toBe("number");
+          expect(article.article_img_url).toMatch(/^https?:\/\/\S+$/);
+          expect(typeof article.comment_count).toBe("number");
+          expect(article.body).toBeUndefined();
+        });
+      })
+    })
+  })
```

Comment:
There's one more test I'd like to see:

- [ ] Status 200: If topic query exists, but there are no matching articles, returns an empty array